tests/constants.py

```diff
@@ -14,7 +14,7 @@
 
 TEST_BUILD = "test-build-123"
 TEST_CANCELLED_BUILD = "test-build-cancel-123"
-TEST_BUILD_CONFIG = "path-master"
+TEST_BUILD_CONFIG = "path-master-....."
```

Comment:
Also needs an equivalent change in tests/fake_api.py to use matching instead of string comparison when looking up URL resources.